README.md

```diff
@@ -1,19 +1,152 @@
+
+* After creating a quiz, access it and press the `Edit quiz` button.
+* On the right side of the page, go to `Add > + a new question` and select the `quizmanager` question type.
+* Select the category where you want your question to be from.
+* Set the difficulty from the given list _(Easy - Hard)_.
+* Set the topic that you want your question to have. Note: topics are case **insensitive**, and you can choose **only one** topic for one question.
+* Press the `Save changes` button.
+* Repeat the instructions for all the questions you want in your quiz.
+
+# Managing Questions
+
+The plugin can use any [question types](https://docs.moodle.org/311/en/Question_types) existent in the question bank, as long as the have the associated difficulty and topic tag.
+
+If questions are imported from an external source, using any [format](https://docs.moodle.org/311/en/Import_questions#Question_import_formats) supported by Moodle, the following rules must be followed:
+* The difficulty tag must be one from the following list: `Easy, Easy-Medium, Medium, Medium-Hard, Hard`.
+* The topic tag can take any form, as long as it is consistent with the set topic in the question edit form (see [Adding Questions](/README.md/#adding-questions)).
+* The `last_used` tag is optional (if not present will be considered 0), and the format is: `last_used:time`, where time is an integer representing time since the Epoch (00:00:00 UTC, January 1, 1970), measured in seconds.
+
+Details for the importing procedure can be found in the [Moodle documentation](https://docs.moodle.org/311/en/Moodle_XML_format).
+
+The following example can be used for importing questions with tags in MXML format (note that you can modify any detail about the questions, the example only shows the use of tags in MXML):
+
+```xml
+<?xml version="1.0" ?>
+
+<quiz>
+    <question type="multichoice">
+        <name>
+            <text>
+                Question Name Goes Here
+            </text>
+        </name>
+        <questiontext format="html">
+            <text>
+		    Question Text Goes Here
+            </text>
+        </questiontext>
+
+        <tags>
+            <tag>
+                <text>
+			First Tag Goes Here
+                </text>
+            </tag>
+            <tag>
+                <text>
+			Second Tag Goes Here
+                </text>
+            </tag>
+	    <tag>
+		<text>
+			Third Tag Goes Here
+		</text>
+	    </tag>
+        </tags>
+
+        <answer fraction="100">
+            <text>
+                Correct Answer Goes Here
+            </text>
+        </answer>
+
+        <answer fraction="0">
+            <text>
+                Incorrect Answer Goes Here
+            </text>
+        </answer>
+    </question>
+</quiz>
+```
+
+# Updating local database
+
+After every question import, the local database that stores the `last_used` tag must be updated, or the questions will be used with the default value of 0, making it useless.
```

Comment:
```suggestion
After every question import, the local database that stores the `last_used` tag must be updated.
Otherwise the `last_used` tag will be set to `0`, losing its meaning.
```